Content/Items/Vitric/Weapons.MagmiteVacpack.cs

```diff
@@ -1,4 +1,9 @@
-﻿using System;
+
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using StarlightRiver.Content.Items;
+using StarlightRiver.Core;
```

Comment:
Why were these usings added? These should be redundant with the global usings added in the recent refactor